esp-metadata/src/lib.rs

```diff
@@ -276,5 +278,11 @@ fn define_all_possible_symbols() {
             // https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-check-cfg
             println!("cargo:rustc-check-cfg=cfg({})", symbol);
         }
+        for memory in config.memory() {
+            println!(
```

Comment:
This doesn't include ALL memory regions, just the ones the chip defines, right?

Reply:
yesn - the outer loop iterates over all chips so it's the all regions defined for any chips